buildSrc/src/main/kotlin/com/datadog/gradle/plugin/srpayloadupdate/SrPayloadOverrideTask.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.gradle.plugin.srpayloadupdate
+
+import org.gradle.api.tasks.Exec
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputDirectory
+
+open class SrPayloadOverrideTask : Exec() {
+
+    @OutputDirectory
+    val payloadUpdateDirectory = project.file("./src/androidTest/assets/session_replay_payloads/")
+
+    @Suppress("MaxLineLength")
+    @Input
+    val payloadInputDirectoryPath: String = "storage/emulated/0/Android/data/com.datadog.android.sdk.integration/cache/session_replay_payloads/."
```

Comment:
```suggestion
    val payloadInputDirectoryPath: String = "storage/emulated/0/Android/data/com.datadog.android.sdk.integration/cache/session_replay_payloads/"
```

Reply:
nope...actually I need the `.` to copy everything recursively in case